core/src/test/scala/org/apache/spark/deploy/security/HadoopDelegationTokenManagerSuite.scala

```diff
@@ -46,6 +47,7 @@ class HadoopDelegationTokenManagerSuite extends SparkFunSuite with Matchers {
     delegationTokenManager.getServiceDelegationTokenProvider("hadoopfs") should not be (None)
     delegationTokenManager.getServiceDelegationTokenProvider("hbase") should not be (None)
     delegationTokenManager.getServiceDelegationTokenProvider("hive") should not be (None)
+    delegationTokenManager.getServiceDelegationTokenProvider("kafka") should not be (None)
```

Comment:
There are other test methods where all the others DT providers are tested similar to this. 
Like `test("using deprecated configurations")` you can extend them with this line as well (to keep the symmetry).

Reply:
Fixed.